README.md

```diff
@@ -154,6 +154,11 @@ A list of optional component that are not included in HELICS but are optionally
 
```

Comment:
Should we get rid of the `### General HELICS` heading and just list the papers? Doesn't seem like it makes sense to categorize them unless there are a lot of papers.

Reply:
I made the change to get rid of that header and your other suggestion